src/components/AccessibilityWidget/accessibility.css

```
@@ -0,0 +1,219 @@
+ *
+ * Aplica uma stack de fontes com OpenDyslexic primeiro (caso o app
+ * consumidor a tenha carregado), com Comic Sans MS como fallback —
+ * disponível em quase todos os SOs e recomendada por orgãos como a
+ * British Dyslexia Association como fonte de leitura acessível.
+ */
+
+html.a11y-dyslexia-font,
+html.a11y-dyslexia-font body,
+html.a11y-dyslexia-font body * {
+  font-family:
+    'OpenDyslexic', 'Comic Sans MS', 'Atkinson Hyperlegible', Verdana,
```

Stylelint (17.10.0)

[error] Expected no quotes around "OpenDyslexic" (font-family-name-quotes)

(font-family-name-quotes)